Automattic/pocket-casts-ios
# Playback 2024: Epilogue Screen

| 📘 Part of: #2250 |
|:---:|

Completes #2323

https://github.com/user-attachments/assets/0eb32463-31e1-4c41-836e-5ec9b58bdf3a

## To test

* Enable the `endOfYear2024` feature flag
* Restart the app
* Tap the "Playback 2024 is here!" cell
* Tap through the stories until you get to the final one and verify it looks correct
* Tap the "Play again" button and make sure it restarts

## Checklist
